Gandalan.IDAS.WebApi.Client/DTOs/Nachrichten/NachrichtenDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Gandalan.IDAS.WebApi.Client.DTOs.Nachrichten
+{
+    public class NachrichtenDTO
+    {
+        public Guid NachrichtGuid { get; set; }
+        public Guid MandantGuid { get; set; }
+        public Guid BesitzerMandantGuid { get; set; }
+        public string Context { get; set; }
+        public string Nachricht { get; set; }
+        public bool IstAktiv { get; set; }
+        public DateTime? GueltigAb { get; set; }
+        public DateTime? GueltigBis { get; set; }
+
+        public bool IsValid()
+        {
+            var now = DateTime.Now;
```

Comment:
Are you sure `GueltigAb` and `GueltigBis` are not stored as UTC? If it is UTC, you should use `UtcNow` instead of `Now`

Reply:
![image](https://user-images.githubusercontent.com/115798661/215061981-40358334-81b5-4f68-93ae-1bd6d79d5a13.png)

Good hint.
From the database it returns a non zulu date. Should be "27.01.2023Z00:00:00" for Utc? So `.now` should be right,  or am i missing something?

Maybe we should convert it to zulu when saving the messages in Notify?

Reply:
I can't see `Nachrichten` in `IBOS3Context`. Where is it?

Reply:
We need to check whether or not `IDASNotify` uses UTC, but it should do that. Otherwise we might have unexpected behaviour all around

Reply:
So i tried to implement saving and loading IDASNotify-NachrichtenDTO with UTC, but found something strange:
When saving the message the debugger shows correctly that the date is specified as UTC.
![image](https://user-images.githubusercontent.com/115798661/215122314-80e8f4fb-9854-480a-9bda-09d8edbe8e46.png)
 
But in the database it is written without the "Z" extension.
![image](https://user-images.githubusercontent.com/115798661/215121682-c471e459-2638-439c-a681-31b14d3c9b53.png)

And when loading the messages again i says Kind "Unspecified"
![image](https://user-images.githubusercontent.com/115798661/215121499-d0d9baa1-b8d0-4732-9039-de0423777567.png)

Seems like it needs to be specified somewhere, anybody knows where?

Reply:
Check API caller for `IDASNotify.Controllers.NachrichtenKonfigController.Put()`

Reply:
Do we really should use `DateTime.NowUtc`? I saw that the client code uses `DateTime.Now` for checkin dates, too.

Maybe we could assume, that Producer and Clients are in the same timezone? The notifications are only set for specific days, so it doesn''t need to be accurate to the hour? Do i see that right?

Reply:
About 1 or 2 years ago, I modified a lot of DateTimes properties to UTC. Few of them were reverted by @gdl-gruendig back to local timezone (mostly for frontend/I3 Client), but I think this was because of "not synchronized" already existing display logic. For new properties we should use UTC IMO

Reply:
I have implemented UTC and updated the microservice, too.
Thanks for your input.